modules/configuring-cluster-loader.adoc

```diff
@@ -0,0 +1,233 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/using-cluster-loader.adoc
+
+[id='configuring_cluster_loader_{context}']
+= Configuring Cluster Loader
+
+Create multiple namespaces (projects), which contain multiple templates or pods.
```

Comment:
s/Create/You can create ??

Reply:
Should you s/namespaces (projects)/projects (namespaces)? Or, are moving the 4.0 docs to use _namespace_ over _project_?